ext/random/tests/01_functions/random_bytes_error.phpt

```diff
@@ -3,20 +3,21 @@ Test error operation of random_bytes()
 --FILE--
 <?php
 //-=-=-=-
-
+var_dump(random_bytes(0));
```

Comment:
This should be moved to a separate test.

Reply:
how about moving it to `ext/random/tests/01_functions/random_bytes.phpt` ? the description is
```
--TEST--
Test normal operation of random_bytes()
--FILE--
```
and it already has a
```
var_dump(strlen(bin2hex(random_bytes(16))));
```
we could add a
```
var_dump(strlen(random_bytes(0)));
```
expecting a `int(0)`

Reply:
update: moved it to ext/random/tests/01_functions/random_bytes.phpt